Sources/Stagehand/AnimationCurve/AnimationCurve.swift

```diff
@@ -27,4 +27,6 @@ public protocol AnimationCurve {
 
     func adjustedProgress(for progress: Double) -> Double
 
+    func rawProgress(for adjustedProgress: Double) -> [Double]
```

Comment:
Probably worth a comment explaining why `adjustedProgress(for:)` returns a `Double`, but `rawProgress(for:)` returns a `[Double]`.

```suggestion
    /// The raw (uncurved) progress values that correspond to the specified adjusted (curved) progress.
    ///
    /// Note that unlike the raw -> adjusted calculation that always results in one value, there may be multiple raw
    /// values corresponding to one adjusted value. In other words, X(t) is monotonic, but Y(t) is not.
    func rawProgress(for adjustedProgress: Double) -> [Double]
```